### ISSUE TITLE
The GUI is not properly accessible on Windows

### ISSUE THREAD

**Issue by sariths** [IGD-Labs/Groundhog#3]
Hi @germolinal , I was trying out the latest release and found that the GUI was mangled and unresponsive on Windows. 
![9-22-2016 12-49-12 am](https://cloud.githubusercontent.com/assets/10361848/18737383/8574f166-805e-11e6-9382-546cabcfc77c.png)

I am on a 64-bit Windows 7 machine and tried the extension on Sketchup 2014 and 2016. The installation was very smooth and I did not get any errors.

**Comment by sariths** [comment]
Quick update: 
I tried keeping the Ruby Console open to see if it was generating any errors. It does not seem to be generating any errors. 
Secondly, I tried out your example files and they seem to export fine as well.

**Comment by germolinal** [comment]
Hi Sarith, thanks for reporting this bug. 

I work on Windows myself, and everything runs smooth. The error you mentioned should be related to the WebDialogs, which are html/JavaScript/CSS files, opened with Internet Explorer on Windows and Safari on Mac. This new version 0.9.2 is using JqueryUI. 

Is it possible for you to go to jqueryui.com using your Internet Explorer and see if everything works?

PS: I am unavailable until October 10, so please be patient.

**Comment by sariths** [comment]
Hi German,

If by everything works you mean if the website loads or not, then everything is working in that regard.
![9-22-2016 3-36-05 pm](https://cloud.githubusercontent.com/assets/10361848/18763232/656e0f3e-80da-11e6-9a1e-49e21041d625.png)
My Internet Explorer is version 10 and I am running a Windows 7 SP1 OS. I did try out the plugin on my Windows 10 Surface Pro 4 just to see if the issue was specific to my machine. However, I got the same error there as well.

Take your time :) . I was just interested in trying out the new electric lighting features that you implemented recently.

**Comment by germolinal** [comment]
Sarith,

I am back... not really working on the issues, but I will soon.

did you try updating your browser to Internet Explorer 11? Any reason not
to do that? That is the version I am using, and I do not seem to find IE10
for Windows 10.

Best,

Germán

2016-09-22 16:40 GMT-03:00 Sarith Subramaniam <notifications@github.com>:

**Comment by sariths** [comment]
Hi Germán,

 

 

No, I haven’t tried IE11.  I think one of the reasons I couldn’t do IE11 was that my Windows 7 installation was too outdated. 

 

I will check on this on report back to you within this week. Thanks for keeping me posted!

 

 

Regards,

Sarith

**Comment by germolinal** [comment]
Hello Sarith, 

I found a second person with the same error yesterday. Unfortunately, I did not manage to find the error (debuging webdialogs is somehow hard on SketchUp). I will try to prepare a file that may opened on IE10 or IE11... are you willing to help me test it? I understand if you are not.

Kind regards,

Germán

**Comment by sariths** [comment]
Hi German,

I apologize for not getting back to you sooner. I have been crazy busy the past month. I have a Windows 10 installation on my surface pro. I will let you know by tomorrow ( for sure this time !).

Sarith

**Comment by sariths** [comment]
Hi German,

I just checked the installation with IE11 on Windows 10. The problem still persists.  I am attaching a screenshot below
![capture](https://cloud.githubusercontent.com/assets/10361848/19754156/eeb1f13e-9bd8-11e6-8b22-b604e2132d4b.PNG)

**Comment by germolinal** [comment]
That is not good.... let me think about the possible causes. I will let you
know!

**Comment by germolinal** [comment]
Hello @sariths , 

I figured out how you may help me debug this... is it possible that you comment [THIS ](https://github.com/IGD-Labs/Groundhog/blob/master/IGD_Groundhog/src/html/design_assistant.html#L385) line in the Design Assistant file and then open the design_assistant.html open it on IE11? 

I am interested on the messages that may appear in the console... as I think this is a javascript or CSS issue.

Thanks!

**Comment by sariths** [comment]
Hi German,

 

 

I apologize for the late reply. I have been away for a while and haven’t been active on the internet for the last week or so. I will get back to you with the test results within this week.

 

 

Regards,

Sarith

**Comment by germolinal** [comment]
Made some important changes in UI... maybe they fixed this...????

Best!

PS: SketchUp 2017 seem to have its own Chromium-based UI windows! this should eliminate issues.

**Comment by sariths** [comment]
Hi @germolinal, I was supposed to get back to you with the test but completely forgot about it :( . Anyways, I finally got around to testing the new release (downloaded from https://github.com/IGD-Labs/Groundhog/raw/master/Groundhog_win64.rbz) and everything is working well on that. 
The GUI works without any glitches and I also did a simple export.

![11-26-2016 5-46-25 pm](https://cloud.githubusercontent.com/assets/10361848/20644390/92605994-b400-11e6-8cb1-35401f5239a1.png)

Thank you for addressing this. Since this issue is fixed, I am closing it.
